### DISCUSSION TITLE
How to bypass row-click event for some columns in QTable?

### DISCUSSION THREAD

**Discussion by meo1** [quasarframework/quasar#14102] [General - Components / Directives / etc] [answered]
How to bypass row-click event for some columns in QTable?

https://codepen.io/meo1meo1/pen/GRxygNX?editors=1011

**Comment by MilosPaunovic** [comment]
By conditionally checking your condition. If you don't provide more information, that's as much of the help you could get.

**Reply by meo1** [reply]
I am sorry Milos, I have forgot to save codepen code of mine. I hope this time it is understandable:

https://codepen.io/meo1meo1/pen/GRxygNX?editors=1011

I do not want to row-click event fires when I click edit or delete actions.

**Comment by MilosPaunovic** [comment, marked as answer]
Use event modifiers, `@click.stop="deleteRow(props)"`.

**Comment by meo1** [comment]
I am sorry I could not explain myself correctly. I do want to "edit" and "delete" @click events fires but I do not want to "youClickMe" @row-click event fires when I click "edt" and "delete" icons. It is just the opposite of your solution.

**Comment by MilosPaunovic** [comment]
Have you tried adding modifier to edit & delete click events?
[https://codepen.io/milospaunovic/pen/YzaYyaZ?editors=1011](https://codepen.io/milospaunovic/pen/YzaYyaZ?editors=1011)

When you click row, it triggers `youClickMe` method. When you click edit button, `editRow` is triggered. When you click delete, `deleteRow` is triggered.
